src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -564,12 +565,18 @@
   --#{$menu-toggle}__controls--MarginRight: var(--#{$menu-toggle}__button__controls--MarginRight);
   --#{$menu-toggle}__controls--MarginLeft: var(--#{$menu-toggle}__button__controls--MarginLeft);
 
+  display: inline-flex;
+  align-items: baseline;
   align-self: var(--#{$menu-toggle}__button--AlignSelf);
   padding-inline-start: var(--#{$menu-toggle}__button--PaddingLeft);
   padding-inline-end: var(--#{$menu-toggle}__button--PaddingRight);
   color: inherit;
   background-color: var(--#{$menu-toggle}__button--BackgroundColor);
   border: 0;
+
+  &.pf-m-text {
```

Comment:
Doesn't look like the padding-inline-start is being applied because of this rule having higher specificity https://github.com/patternfly/patternfly/blob/313313390fe54fc903fc1f9a60e177b745dd1364/src/patternfly/components/MenuToggle/menu-toggle.scss#L395-L399

One way you could make it work is leave the existing `padding-inline-start` that you added, and also declare this var

```
--#{$menu-toggle}--m-split-button--last-child--PaddingLeft: var(--#{$menu-toggle}__button--m-text--PaddingInlineStart);
```